packages/manager/src/features/Billing/BillingPanels/BillingActivityPanel/BillingActivityPanel.tsx

```diff
@@ -365,37 +350,39 @@ export const BillingActivityPanel = (props: Props) => {
               </div>
             )}
             <div className={classes.flexContainer}>
-              <Select
+              <Autocomplete
+                onChange={(_, item) => {
+                  setSelectedTransactionType(item?.value ?? 'all');
+                  pdfErrors.clear();
+                  pdfLoading.clear();
+                }}
                 value={
                   transactionTypeOptions.find(
-                    (thisOption) => thisOption.value === selectedTransactionType
+                    (item) => item.value === selectedTransactionType
                   ) || null
                 }
                 className={classes.transactionType}
-                hideLabel
-                inline
-                isClearable={false}
-                isSearchable={false}
                 label="Transaction Types"
-                onChange={handleTransactionTypeChange}
+                multiple={false}
                 options={transactionTypeOptions}
-                small
+                placeholder=" "
               />
-              <Select
+              <Autocomplete
```

Comment:
Should we hide the label, remove padding, and disableClearable to keep parity with the current UI?

```suggestion
              <Autocomplete
                disableClearable
                textFieldProps={{
                  hideLabel: true,
                  noMarginTop: true,
                }}
```

Reply:
I updated the styles to match the other panels in the page, which removed extra white space around the Header and outline the panel. The labels for the select components are there to improve a11y (waiting feedback from UX). I added the `disableClearable` prop.